### PR TITLE
Home page php

### PR DESCRIPTION
**Pull request by LewisWelch94**

### CODE REVIEW

**Comment by OnikNoor97** [comment]
Going to clean this up before merging, approach is fine, however. Too many echo is a bad idea

File: Home/index.php [hunk 1]
```diff
@@ -1,19 +1,25 @@
...
 
     <body>
-        <?php include("../Global/navBar.php"); ?>
-        <?php include("../Home/homeMessage.php"); ?>
-        <h1>This page is under maintenance</h1>
-        <?php include("../Global/scripts.php"); ?>
+
+    <?php
+    if ($userLoggedIn == null) include("aboutUs.php"); else include("dashboard.php");
+    ?>
```
Comment (OnikNoor97) [inline Home/index.php:17]: Put it all in one line for clean up

File: Home/index.php [hunk 2]
```diff
@@ -1,19 +1,25 @@
...
+
     </body>
+    <footer> 
         <?php include("../Global/editUserModal.php"); ?>
-
+        <?php include("../Global/scripts.php"); ?>
+        <link rel="stylesheet" href="../Css/homePage.css">
```
Comment (OnikNoor97) [inline Home/index.php:23]: Stylesheets should be in the top, only JS should be in the bottom

File: User/userController.php [hunk 3]
```diff
@@ -246,6 +246,6 @@ public function loadDropdownItems($userLoggedIn)
 			echo "<a class='dropdown-item' id='logoutNav' href='../User/logout.php'>Logout</a>";
 			if($userLoggedIn->getLevel() > 3) echo "<a class='dropdown-item' id='adminNav' href='../User/admin.php'>Admin</a>";
 		}
-	}
+	}
```
Comment (OnikNoor97) [inline User/userController.php:249]: Clean this up

**Review by OnikNoor97 — COMMENTED** [review]
Leaving the CSS changes to Tufan

File: Home/dashboard.php [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+                <hr class="small-hr">
+                <ul class="list-inline">
+                    <li class="list-inline-item">Lorem ipsum</li>
+                    <li class="list-inline-item">Phasellus iaculis</li>
+                    <li class="list-inline-item">Nulla volutpat</li>
+                </ul>
+            </span>
+        </div>
+    </div>
+</div>
```
Comment (tufanbutuner) [inline Home/dashboard.php:25]: Looks good, but the 'Your projects' section is coming off the page. I have attached a screenshot below.
Screenshot 2020-12-23 at 19 27 27

**Review by tufanbutuner — COMMENTED** [review]
Great work, the home page looks much cleaner.

**Pull request merged**